### DISCUSSION TITLE
Create request failed, and the error was reported as follows, Can anyone help me solve it?

### DISCUSSION THREAD

**Discussion by sunxingdi** [RequestNetwork/requestNetwork#1325] [Questions]
const request = await requestClient.createRequest(requestCreateParameters);

When I execute this code, I get the following error. Can anyone help me solve it?

Error: Transaction confirmation not received. Try polling
            getTransactionsByChannelId() until the transaction is confirmed.
            deferDelay: 3000ms,
            maxRetries: 30,
            retryDelay: 1000ms,
            exponentialBackoffDelay: 0ms,
            maxExponentialBackoffDelay: 30000ms
    at E:\Code\request-network-interaction\node_modules\@requestnetwork\request-client.js\src\http-data-access.ts:132:19
    at processTicksAndRejections (node:internal/process/task_queues:95:5)

**Comment by MantisClone** [comment]
Hi, @sunxingdi This error indicates that the transaction confirmation between the Request Client and the Request Node timed out.

We've received reports from other builders that this error is occurring and we're working on a few fixes:

In the meantime, you can try to adjust the `httpConfig.getConfirmation...` settings on your `RequestNetwork` instance to increase the timeout interval.

```ts
const requestNetwork = new RequestNetwork({
    nodeConnectionConfig: {
      baseURL: https://xdai.gateway.request.network/,
    },
    signatureProvider,
    httpConfig: {
      getConfirmationMaxRetry: 60 // (default = 30)
    },
  });
  ```
